### PR TITLE
Stocks Resampling Demo

### PR DESCRIPTION
**Pull request by fpliger**

### CODE REVIEW

**Comment by bryevdv** [comment]
Hi @fpliger I ran all the demos here and they seem to work great. I think having a `README` with lots of sub-demos in it is way too complicated, however. In fact, I think basically we need to standardize a simple convention that *all* demos can adhere to. Something like a `run.py` the top level, that takes a subdirectory as an argument, and runs and all demos in that subdirectory. It would open a landing page that has links to the demo page or pages. If no argument is given, runs *all* the demos in all the subdirs, and the landing page lists links to all the demo pages. 

So basically each demo subdir is going to need some kind of metadata (maybe a yam file?) that describes: processes that need to be run, all the links that should get added to the landing page, eventually could include screenshots and extended descriptions for each demo. I have talked with @Maggie-M about possibly working on this demo runner since it is similar to the interactive tester in some ways. 

With that in mind, we could go ahead and merge this and make the demo-runner a separate PR.

**Comment by fpliger** [comment]
I'm +1 on merging this one as is and working on the  #4 separately . Same applies for improvements to this demo. There are quite a few but I see more value merging sooner (now is better then never :-)

**Comment by bryevdv** [comment]
Sounds good to me.

**Comment by fpliger** [comment]
@bokeh/core I'll be merging this later today (leaving the demo runner integration for a later PR). Please let me know if you have reasons for not merging it.

**Comment by damianavila** [comment]
:+1:
